tests/LSP16UniversalFactory/LSP16UniversalFactory.test.ts

```diff
@@ -512,7 +512,7 @@ describe("UniversalFactory contract", () => {
         await context.universalFactory.deployCreate2Init(
           UPBytecode,
           salt,
-          "0xaabbccdd",
+          "0x00000000aabbccdd",
```

Comment:
We could add an extra comment here to describe that it is just some random data sent along.

```suggestion
            "0x00000000aabbccdd", // send some random data along prepended with `
0x00000000`
```